src/modules/uORB/Subscription.hpp

```diff
@@ -104,7 +166,7 @@ typedef SubscriptionBase SubscriptionTiny;
 /**
  * The subscription base class as a list node.
  */
-class __EXPORT SubscriptionNode : public SubscriptionBase, public ListNode<SubscriptionNode *>
+class  SubscriptionNode : public SubscriptionInterval, public ListNode<SubscriptionNode *>
```

Comment:
While at it, can you remove the linked list?

Reply:
Yes I'd like to, but need to review existing uses (mostly LPE at this point).